src/app/pages/privacy-policy/privacy-policy.component.html

```diff
@@ -0,0 +1,45 @@
+<app-hill-background cover></app-hill-background>
+<div class="home-container">
+  <mat-card class="column">
+    <mat-card-title>
+      <h1 style="font-size:60px;" class="title">Privacy Policy</h1>
+    </mat-card-title>
+    <mat-card-content>
+        <h1>Collection of Information</h1>
+          <h2>Information you give</h2>
+            <p>Buzz About receives personal information when you log in using a Google account. This includes your email address.</p>
+          <h2>Information we collect automatically</h2>
+            <p>We collect the information about your interactions on the BuzzAbout site. While Buzz About itself does not collect any information automatically about your machine, third party services that we use may collect that information. Check the privacy policies of our third party services below to read about the information they collect.</p>
+        <h1>Use of Information</h1>
+          <p>The data collected by Buzz About is used by a research team at the University of Minnesota - Morris. The purpose of this research is to learn more about how taking part in activities that show what bees do and what they need might help people understand more about how to support bees and how they help people. We want to know more about how we can use technology to help people learn more from these activities. The results of this research might help people design technology in a way that helps people learn more effectively.</p>
+          <p>In line with ethical guidelines for research, efforts will be made to limit the use of any data collected by Buzz About, including personal information, to the people who need to view the information. This may include the University of Minnestoa’s Institutional Review Board and other members of the institution.</p>
+        <h1>Third-Party Services</h1>
+          <p>Buzz About uses a third party service Google Firebase to authenticate users and to store our data. You can access Google Firebase’s privacy policy <a href = "https://firebase.google.com/support/privacy">here.</a></p>
+        <h1>Data Security</h1>
+          <p>Buzz About makes reasonable efforts to avoid the loss, misuse, or unauthorized access, disclosure, alteration or destruction of your data, including personal information. Users should understand that no data storage can guarantee 100 percent security. Because our data is stored using a third party, please reference the Google Firebase privacy policy for more information on how your information may be secured. Again, we cannot guarantee the security of third party services.</p>
```

Comment:
> Again, we cannot guarantee the security of third party services. 

I don't think this is necessary.

Reply:
Looking at other examples, that specific wording seemed to be important.